src/oph/ehoks/db/db_operations/db_helpers.clj

```diff
@@ -188,12 +188,14 @@
 
 (defn from-sql
   "Convert maps returned by database functions to format expected elsewhere."
-  ([m operations]
-    (-> (convert-sql m operations)
-        remove-nils
-        remove-db-columns
-        to-dash-keys))
-  ([m] (from-sql m {})))
+  ([m operations keep-columns]
+    (let [remove-cols-func #(remove-db-columns % keep-columns)]
+      (-> (convert-sql m operations)
+          remove-nils
+          remove-cols-func
```

Comment:
This is not an issue per se, but you can remove the `remove-cols-fund` `let` binding and put `(remove-db-columns keep-columns)` directly to the place of `remove-cols-func` at line `195`.

Reply:
Good catch! That `remove-cols-func` variable contained a conditional value earlier, but I forgot to inline it when the conditional logic was removed.